addons/prometheus/0.38.x/prometheus-13.yaml

```diff
@@ -110,6 +110,19 @@ spec:
               - path: /api/v1/metrics/prometheus
                 port: metrics
                 interval: 30s
+          - name: kubeaddons-service-monitor-api-v1-metrics-prometheus-http-10s
```

Comment:
How about removing the `10s` from the def for this service monitor (including the `servicemonitor.kubeaddons.mesosphere.io/interval: "10s"` label)? in case we want to use this service monitor for other services with `api/v1/metrics/prometheus` metrics path and `http` endpoint. It also avoids us having to also modify 2 addon configs if we were to ever want to change the interval

Reply:
We should go away from the need to manage Service Monitors in external addons not close to the addon which should be monitore. At the moment I would say we leave as is and we will see what comes next and how often we need to touch the Service Monitors here again. At best we can get rid of managing it here 😉